src/io/sqlite_store/mod.rs

```diff
@@ -183,6 +229,15 @@ impl KVStore for SqliteStore {
 	) -> io::Result<()> {
 		check_namespace_key_validity(primary_namespace, secondary_namespace, Some(key), "write")?;
 
+		if self.config.transient_graph
+			&& primary_namespace == NETWORK_GRAPH_PERSISTENCE_PRIMARY_NAMESPACE
+			&& secondary_namespace == NETWORK_GRAPH_PERSISTENCE_SECONDARY_NAMESPACE
+			&& key == NETWORK_GRAPH_PERSISTENCE_KEY
+		{
+			// Alby: transient network graph is not persisted.
```

Comment:
Could you add a debug log here so we can test and ensure it does not try to write more than once